dashboard/app/views/peer_reviews/show.html.haml

```diff
@@ -20,7 +20,6 @@
     %p Please review the peer submission in the grey box below.
 
   .submitted_readonly{style: 'padding: 8px;'}
-    %script{src: minifiable_asset_path('js/initApp/initApp.js')}
```

Comment:
Any idea how initApp was being used outside of the `_apps_dependencies` context?